11.0.Dockerfile

```
@@ -114,11 +114,13 @@ RUN mkdir -p auto/addons auto/geoip custom/src/private \
 COPY qa /qa
 RUN python -m venv --system-site-packages /qa/venv \
     && . /qa/venv/bin/activate \
+    # HACK: Upgrade pip: higher version needed to install pyproject.toml based packages
+    && pip install -U pip \
     && pip install --no-cache-dir \
         click \
         coverage \
         flake8 \
-        pylint-odoo \
+        git+https://github.com/Tecnativa/pylint-odoo.git@migrate-poetry \
```

Please change this, so it is a better reminder that tells us this should be undone when that PR is merged:

```suggestion
        git+https://github.com/OCA/pylint-odoo.git@refs/pull/329/head \
```